app/views/helpers/taxsInputTypeRadioGroup.scala.html

```diff
@@ -118,7 +118,7 @@
     }
 }
 
-<fieldset @fieldSetClass @fieldSetAttributes role="radiogroup" aria-required="true" @if(fieldErrors.nonEmpty) {aria-labelledby="@params.field.name-error-0"}>
+<fieldset @fieldSetClass @fieldSetAttributes @if(fieldErrors.nonEmpty) {aria-describedby="@params.field.name-error-0"}>
         @legend
         @formHint
```

Comment:
If there is a hint you want this in the aria-describedby too

Reply:
no hint. Just H1 followed by radio buttons